app/api/csv/specs/csvLoaderSelects.spec.js

```diff
@@ -9,7 +9,7 @@ import { CSVLoader } from '../csvLoader';
 
 const fixtureFactory = getFixturesFactory();
 
-const commonTranslationContexts = (id1, id2) => [
+const commonTranslationContexts = (id1, id2, id3) => [
```

Comment:
@LaszloKecskes Probably not related to this PR, but if I delete all the commonTranslationContexts except system, the tests continue passing.

Reply:
That one's tricky, since it is something that we did not test first and it is a smoke test for an issue we had during writing. 
We have assumed that the thesaurus saving will take care of the translations. However, there was an issue with concurrency: we are saving multiple thesauri, and when we did it in an async manner, they have overwritten each other's changes to the translations, and only the last one got entered properly. So, for the test I have decided to have a "translations" structure as close to the real one as possible, and then test that the changes go through as they should.
However, since this was a smoke test, and the related problem was taken care of (this is the reason for the somewhat convoluted syncSaveThesauri) , now it seems like this is not testing anything. Nevertheless, I would leave it as is, or just add a comment somewhere (in the test, next to the fixtures, both...) to indicate that this was a smoke test.

Reply:
Ok, it's fine I think. I'll merge it as it is.